src/benchmarks/micro/libraries/Microsoft.Extensions.Configuration/ConfigurationBinderBenchmarks.cs

```diff
@@ -0,0 +1,75 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
```

Comment:
Can we use System.Text.Json instead? No reason to add another dependency on Newtonsoft.

Reply:
Sure. Done.